<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB Shell</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QPushButton" name="execute_button">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="query_text"/>
   </item>
   <item row="2" column="0">
    <widget class="QListWidget" name="query_history"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="db_path">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Selected DB</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="db_select_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Select DB</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
